src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -813,37 +830,41 @@ doLedgerEntry(RPC::JsonContext& context)
     if (!lpLedger)
         return jvResult;
 
-    static LedgerEntry ledgerEntryParsers[] = {
-        {jss::index, parseIndex, ltANY},
-        {jss::account_root, parseAccountRoot, ltACCOUNT_ROOT},
+    static std::array<LedgerEntry, 21> ledgerEntryParsers = {
```

Comment:
You can totally avoid specifying the size here like so: https://godbolt.org/z/6jETseadE
The downside is that you need to wrap each entry with its type (LedgerEntry{...}).

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/5199/commits/c255d460b42ac22e26ad29f94bd80dea97847c49